### PR TITLE
Stand alone science page

### PR DESCRIPTION
**Pull request by snim2**
This PR is a step towards making the Science page more of a one-stop shop. The aim is that people pointed to the page from the SSI blog post or an email won't have to look around the site for programme or registration details. (Thanks to Jonathan Fine for nudging this forward).

It may be that we need to get in touch with Simon about the timing of his keynote before this PR is merged.

It would be good to get the talks listed here too, as soon as we can.

### CODE REVIEW

File: content/science.html [hunk 1]
```diff
@@ -3,13 +3,26 @@
...
+    <li><strong>Saturday</strong>
+    <ul>
+    <li>a keynote from <a href="http://www.open.ac.uk/people/ss27739">Simon Sheridan</a>, a developer on the Ptolemy instrument aboard the Rosetta space mission to comet Churyumov-Gerasimenko ('67P').</li>
+    <li><em>Tutorials</em> on Python suitable for beginner Python programmers where you can learn to automate common data analysis and visualisation tasks</li>
+    </ul>
+    <li><strong>Sunday</strong> talks from other scientists and professional software engineers on Python in science and best practices for developing your own code</li>
+    <li><strong>Monday</strong> a collaborative <em>open science sprint</em> where you can bring along code you are working on and we will put you into teams to help improve it. Good tasks to work on might include:
+    <ul>
+    <li>figuring out whether Python is a good fit for your needs</li>
+    <li>speeding up code your existing code</li>
```
Comment (inglesp) [inline content/science.html]: Too much "code"!

File: content/science.html [hunk 2]
```diff
@@ -18,17 +31,29 @@
     <a href="http://www.nature.com/sdata/">Scientific Data</a>.
     This means that any software or figures you produce on the Monday Sprint can be given a DOI and easily shared with the scientific community.
```
Comment (inglesp) [inline content/science.html:32]: Will scientists know what a DOI is?

File: content/science.html [hunk 3]
```diff
@@ -18,17 +31,29 @@
...
     <br/>
-    Watch this space for more announcements -- there may be prizes :)
+    Watch this space for more announcements -- there may even be prizes :)
```
Comment (inglesp) [inline content/science.html]: We've removed a double hyphen above (line 6) so we should be consistent here!

**Comment by inglesp** [comment]
Looks good -- a handful of minor suggestions.

File: content/science.html (hunk 2 above)
Comment (snim2) [inline content/science.html:32]: Yes!

File: content/science.html (hunk 3 above)
Comment (snim2) [inline content/science.html]: Thanks

File: content/science.html (hunk 1 above)
Comment (snim2) [inline content/science.html]: Thanks

**Pull request merged**